examples/09_Kotlin_JS/03_external.md

```diff
@@ -14,8 +14,8 @@ fun main() {
 
 </div>
 
-1. Declare an existing JavaScript function `alert` which takes a single `String` argument
-2. Use it as if it was regular Kotlin.
+1. Declaring an existing JavaScript function `alert` which takes a single `String` argument.
+2. Using `alert` as if it was regular Kotlin.
 
 Note that Kotlin checks during compilation that exactly a single argument of type `String` is passed.
 That prevents a number of bugs even when using pure JavaScript API - same as with regular Kotlin.
```

Comment:
It prevents some bugs even when using pure JavaScript API - it's the same as with regular Kotlin.

actually, I don't really know what the "same as with regular Kotlin" means here??
Maybe there is a clearer way to say it.

Reply:
I'll remove with part. The previous sentence says enough about argument checking and it's common for Kotlin.